README.md

```diff
@@ -214,8 +214,22 @@ elb:
         operator: "=="
         value: 0   
 ```
+4. Find Application ELB's that had zero traffic (requests) in the last week. 
 
-4. Find a difference of more than 10% between DynamoDB Provisioned RCUs to Consumed RCUs. 
+```yaml
+elbv2:
+        - description: Application Loadbalancer ActiveConnectionCount
+          metrics:
+          - name: ActiveConnectionCount
+              statistic: Sum
+          period: 24h 
+          start_time: 168h # 24h * 7d 
+          constraint:
+          operator: "<"
```

Comment:
In `config.yaml` the operator is `==`, can you alignment the README.md file?

Reply:
Updated